docs/source/models/concept_models/vivarium_mncnh_portfolio/ai_ultrasound_module/module_document.rst

```diff
@@ -179,19 +196,29 @@ We assume 100% of ultrasounds are standard (and 0% are AI-assisted) at baseline.
   * - Output
     - Value
     - Note
-  * - A. Ultrasound coverage
+  * - A. Type of ultrasound received
     - *none* / *standard* / *AI-assisted*
     - Used for V&V and for estimation of output B
-  * - B. "Believed" gestational age
+  * - B. Estimated gestational age
     - Point values in days
-    - Used for V&V and facility choice module of intrapartum component
+    - Used for V&V, calculation of output C, and determination of
+      eligibility for antenatal corticosteroids
+  * - C. Believed preterm status
+    - *believed preterm* / *believed term*
+    - Used for V&V and for facility choice module in intrapartum component
 
 
 3.0 Assumptions and limitations
 ++++++++++++++++++++++++++++++++
 
 * The timing of ANC visits impacts the ability to accurately estimate gestational age, but we use an average instead. 
-* The current version of the model does not include any false positive rates for pre-term or LBW. Since a false positive is unlikely to cause harm, only inclusion in higher level care, this seems sufficient. 
+* The current version of the model does not include any false positive
+  rates for pre-term or LBW. [[Is this still true? Since the estimated
+  GA can be greater than or less than the true GA, we'll get some false
+  positives for preterm. And we're not doing anything with LBW as far as
+  I know...]] Since a false positive is unlikely to
```

Comment:
Does anyone remember why we had this comment about not including false positive rates in here? Maybe it was referring specifically to LBW, which we decided not to use for now? @SylLutze @pletale @aflaxman 

If I'm thinking about things correctly, I think we *will* have false positives for preterm (e.g., the true GA is 37.5 weeks, but the estimated GA is 36.5 weeks after adding a random error). Does that seem right?

Reply:
It looks like it goes back to several months ago, so perhaps it predates the error model.

Reply:
https://github.com/ihmeuw/vivarium_research/blame/be84ec20ec17e24a335c375695c38f89cb57ebb9/docs/source/models/concept_models/vivarium_mncnh_portfolio/ai_ultrasound_module/module_document.rst#L202

Reply:
Maybe I'll just delete the reference to pre-term, so that this is only referring to LBW.